components/Grid/Grid.tsx

```diff
@@ -50,7 +50,7 @@ export const Grid = ({
   <AsComponent
     {...props}
     className={cnb(
-      'grid overflow-hidden',
```

Comment:
For `position: sticky` to work, the ancestor containers can't have `overflow:hidden`